lib/elixir/lib/file.ex

```diff
@@ -829,20 +836,21 @@ defmodule File do
   success, `files_and_directories` lists all files and directories copied in no
   specific order. It returns `{:error, reason, file}` otherwise.
 
-  Note: The command `cp` in Unix systems behaves differently
-  depending if `destination` is an existing directory or not.
-  We have chosen to explicitly disallow this behaviour.
+  Note: The command `cp` in Unix systems behaves differently depending on
+  whether `destination` is an existing directory or not. We have chosen to
+  explicitly disallow this behaviour. If `source` is a `file` and `destination`
+  is a directory, `{:error, :eisdir}` will be returned.
 
   ## Examples
 
       # Copies file "a.txt" to "b.txt"
       File.cp_r("a.txt", "b.txt")
 
       # Copies all files in "samples" to "tmp"
-      File.cp_r("samples", "tmp")
+      File.cp_r("samples/", "tmp/")
```

Comment:
I think your goal here is to make the intent clearer but `/` is not strictly necessary, so it can also be confusing. So I would suggest removing the trailing `/` OR have an example with both.

Reply:
I'd rather remove the slashes than have two examples meaning the same.

Reply:
done